services-camel/src/test/java/edu/unc/lib/dl/services/camel/solrUpdate/CdrEventToSolrUpdateProcessorTest.java

```diff
@@ -89,22 +84,25 @@ public void init() throws Exception {
         initMocks(this);
 
         processor = new CdrEventToSolrUpdateProcessor();
-        processor.setJmsTemplate(jmsTemplate);
+        processor.setIndexingMessageSender(messageSender);
 
         targetPid = PIDs.get(UUID.randomUUID().toString());
 
         when(exchange.getIn()).thenReturn(msg);
     }
 
+    @SuppressWarnings("unchecked")
     @Test
     public void testNoMessageBody() throws Exception {
         when(msg.getBody()).thenReturn(null);
 
         processor.process(exchange);
 
-        verify(jmsTemplate, never()).send(any(MessageCreator.class));
+        verify(messageSender, never()).sendIndexingOperation(anyString(), any(Collection.class),
```

Comment:
If you use `anyCollectionOf(PID.class)` then you could skip the warning suppression